main/src/library/DelayMap.flix

```diff
@@ -154,17 +154,17 @@ namespace DelayMap {
     ///
     @Experimental @ParallelWhenPure
     pub def count(f: (k, v) -> Bool \ ef, m: DelayMap[k, v]): Int32 \ ef =
-        let f1 = match (k, v) -> f(k, v);
         def c() = foldLeftWithKey((b, k, v) -> if (f(k, v)) b + 1 else b, 0, m);
-        reifyEff(f1) {
-            case Pure(g) =>
+        typematch f {
+            case g: (k, v) -> Bool \ {} =>
```

Comment:
Does k, v here refer to the params in the signature k, v? I assume so, right?

Reply:
Why do you write `\ {}` here when you did not in `DelayList`? 

I guess it makes sense to be explicit and write `\ {}` everywhere?

Reply:
> Does k, v here refer to the params in the signature k, v? I assume so, right?

Yes. No new variables can be introduced in a typematch (except wildcards `_`).



> Why do you write `\ {}` here when you did not in `DelayList`?
> 
> I guess it makes sense to be explicit and write `\ {}` everywhere?

It also nice lines up with `\ ef`.

Reply:
I suggest to write `\ {}` everywhere then! :)